src/ai/backend/manager/idle.py

```diff
@@ -888,7 +886,11 @@ async def get_checker_result(
     ) -> Optional[float]:
         key = self.get_report_key(session_id)
         data = await redis_helper.execute(redis_obj, lambda r: r.get(key))
-        return msgpack.unpackb(data) if data is not None else None
+        if data is not None:
+            if (unpacked := msgpack.unpackb(data)) > 0:
+                return unpacked
+            return None
+        return None
```

Comment:
We can simplify this like:
```suggestion
        if data is not None and (unpacked := msgpack.unpackb(data)) > 0:
            return unpacked
        return None
```